### DISCUSSION TITLE
Update Discord URL

### DISCUSSION THREAD

**Discussion by tanmaster** [OpnForm/OpnForm#591] [Q&A] [answered]
Hi, I wanted to join the discord server but the invitation link that can be found on github and on https://docs.opnform.com/introduction is expired. Could you update it? 🙏

**Comment by JhumanJ** [comment, marked as answer]
Hi @tanmaster thanks for the headsup, can you please check now?

**Reply by tanmaster** [reply]
Yes! It works now, thank you!
